_includes/replacement.html

```diff
@@ -0,0 +1,14 @@
+{%- if site.data.formula[include.item] %}
+{%- assign r_type = "formula" %}
+{%- else %}
+{%- assign r_type = "cask" %}
+{%- endif %}
+<div>
+<p class="recommended-replacement">Recommended replacement: <strong><a href="{{ site.baseurl }}/{{ r_type }}/{{ include.item | uri_escape }}">{{ include.item | escape }}</a></strong></p>
+{%- if r_type == "cask" %}
+{%- include install_command.html item=include.item modifier=" --cask" %}
+{%- else %}
+{%- include install_command.html item=include.item %}
+{%- endif %}
```

Comment:
Omit the suggested install command. Leave it to the user to follow the link to the replacement and copy the install command from there.